webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -153,4 +153,4 @@ export default class MeetingSettingsModal extends React.PureComponent {
             </Modal>
         );
     }
-}
+}
```

Comment:
Nit: New line at the end of file

Reply:
It doesn't show up a new line in my code though. Also, if you view this file in Github, it doesn't show the new line. Not sure why it is pointing this edit. 
Github is also showing a small red circle with a minus icon which on hover reads "No newline at end of file".